HW1/P3/P3.txt

```diff
@@ -1 +1,2 @@
 # Your discussion here
+('aerst', 11, ['arets', 'aster', 'rates', 'reast', 'resat', 'stare', 'stear', 'strae', 'tares', 'tears', 'teras'])
```

Comment:
Actually, there are 3 different letter combinations that all have 11 acronyms. I am not sure if we were supposed to identify all 3 all just one.

('aerst', 11, ['arets', 'aster', 'rates', 'reast', 'resat', 'stare', 'stear', 'strae', 'tares', 'tears', 'teras'])
 ('aelst', 11, ['least', 'leats', 'salet', 'slate', 'stale', 'steal', 'stela', 'taels', 'tales', 'teals', 'tesla']) 
 ('aeprs', 11, ['asper', 'pares', 'parse', 'pears', 'prase', 'presa', 'rapes', 'reaps', 'spaer', 'spare', 'spear'])